src/core/webidl.js

```diff
@@ -33,6 +33,15 @@ const idlTypedefTmpl = tmpls["typedef.html"];
 // https://github.com/w3c/respec/issues/999
 // https://github.com/w3c/respec/issues/982
 const unlinkable = new Set(["maplike", "setlike", "stringifier"]);
+const topLevelEntities = new Set([
+  "callback interface",
+  "callback",
+  "dictionary",
+  "enum",
+  "interface mixin",
+  "interface",
+  "typedef",
+]);
```

Comment:
We also have `namespace`s, but it seems we don't support namespaces yet! I didn't know that.

Reply:
Let's do that as a new PR.

Reply:
Filed https://github.com/w3c/respec/issues/1795